RileyLinkKit/PumpOpsSynchronous.swift

```diff
@@ -157,8 +149,12 @@ public class PumpOpsSynchronous: NSObject {
     
     let frequencies = [916.55, 916.60, 916.65, 916.70, 916.75, 916.80]
     var results = FrequencyScanResults()
-    
-    defaultWake()
+
+    do {
+      try wakeup()
+    } catch {
+      // Continue anyway
```

Comment:
Yes. If the frequency is too far off, we won't hear the pump's response, but it probably did wake.